internal/services/automanage/arc_machine_automanage_configuration_assignment_resource_test.go

```diff
@@ -0,0 +1,83 @@
+package automanage_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/automanage/2022-05-04/configurationprofilehcrpassignments"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ArcMachineConfigurationAssignmentResource struct{}
+
+func TestAccArcMachineConfigurationAssignment_basic(t *testing.T) {
```

Comment:
Since all of the fields are required and force new, does `basic` make sense here? Should this be `complete` instead?

```suggestion
func TestAccArcMachineConfigurationAssignment_complete(t *testing.T) {
```